tzconvert.py

```diff
@@ -54,9 +65,9 @@ def adjust_datetime(naive_date, city_name, debug=False):
         naive_datetime = current_datetime
         adjusted_datetime = adjust_datetime(naive_datetime, city)
 
-        if not adjusted_datetime:
-            print("Could not fetch adjusted date time")
-            continue
```

Comment:
This continue needs to be present

Reply:
No idea when these lines got removed. I'll add them back.